src/components/export/ExportNotification.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <span></span>
```

Comment:
Yeah, this component doesn't have a template. It's a kind of workaround for Noty not supporting reactive content.

Reply:
Yeah, we'll probably move off-of noty at some point